packages/block-editor/src/components/block-mover/index.js

```diff
@@ -56,12 +56,12 @@ export class BlockMover extends Component {
 		// to an unfocused state (body as active element) without firing blur on,
 		// the rendering parent, leaving it unable to react to focus out.
 		return (
-			<div className={ classnames( 'editor-block-mover block-editor-block-mover', { 'is-visible': isFocused || ! isHidden } ) }>
+			<div className={ classnames( 'editor-block-mover block-editor-block-mover', { 'is-visible': isFocused || ! isHidden, 'is-horizontal': horizontalMover } ) }>
 				<IconButton
 					className="editor-block-mover__control block-editor-block-mover__control"
 					onClick={ isFirst ? null : onMoveUp }
-					icon={ upArrow }
-					label={ __( 'Move up' ) }
+					icon={ horizontalMover ? leftArrow : upArrow }
```

Comment:
We would still need to customize the button labels so I guess customizing the icon does not have a significant increase in the effort.
Also, I expect the design of horizontal movers will be improved and probably it will not just be a rotation.

Reply:
Yes I agree with Jorge, I think rotating won't solve much and this specifying of a separate icon adds future flexibility at little cost.